packages/common/test-utils.js

```diff
@@ -84,3 +86,24 @@ function testAwsClient(Service, options) {
   return new Service(Object.assign(options, { endpoint: 'http://you-forgot-to-stub-an-aws-call' }));
 }
 exports.testAwsClient = testAwsClient;
+
+/**
+ * Create an SQS queue for testing
+ *
+ * @returns {string} - an SQS queue URL
+ */
+async function createQueue() {
+  const createQueueResponse = await aws.sqs().createQueue({
+    QueueName: exports.randomString()
+  }).promise();
+
+  // Properly set the Queue URL.  This is needed because LocalStack always
+  // returns the QueueUrl as "localhost", even if that is not where it should
+  // actually be found.  CircleCI breaks without this.
+  const returnedQueueUrl = url.parse(createQueueResponse.QueueUrl);
+  returnedQueueUrl.host = undefined;
```

Comment:
is this to make it http/https agnostic?

Reply:
This has to do with the way that the `url.format` function determines what host to return.

https://nodejs.org/docs/v6.10.3/api/url.html#url_url_format_urlobject